app/src/main/res/navigation/entrant_graph.xml

```diff
@@ -3,11 +3,26 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/entrant_graph"
-    app:startDestination="@id/nav_home">
+    app:startDestination="@id/nav_browse">
```

Comment:
I missed this before approving but basically these ids are deeply linked with the ids in the menu used for the bottom navigation view. It's in `res/menu`. It's weird but the ids must match exactly and changing it here actually broke the navigation view (but it couldn't be seen in your branch because there was only one root destination).

TL;DR: Keep the top level destinations (home, search, notifications, profile) with matching ids with the menu. Any other destinations, name them anything you like but ideally don't start with `nav_` because `nav_` is designated for top level destinations.

In any case, this is some weird not very well documented stuff that I shouldn't have missed. I'll fix it on my notifications PR.